lesson_6/index6.html

```diff
@@ -16,5 +16,28 @@ <h3>Contact Information</h3>
                 <td>Kyiv, Havela 40</td>
             </tr>
         </table>
+        <br>
+        <form>
+            <fieldset>
+                <legend>Contact form</legend>
+                <label for="email">E-mail<span style="color: red;">*</span>:</label>
+                <input type="email" id="email" name="email" required>
+                <br>
+                <br>
+                <label id="desc">Description:</label>
```

Comment:
label has to have attribute _for_
in other case this tag has no any point to use

Reply:
my bad. i fixed